test/test_oauth2.py

```diff
@@ -211,14 +211,14 @@ def test_it_exchanges(generate_random_string: MagicMock, test_client: FlaskClien
     generate_random_string.return_value = '1a2b3c4e'
 
     with requests_mock.Mocker() as mocker:
-        def token_text(request: Request):
+        def token_text(request: PreparedRequest):
             return urlencode({
                 'client_id': 'server_client_id',
                 'client_secret': 'server_client_secret',
                 'redirect_uri': 'http://localhost/oauth2/check',
                 'grant_type': 'authorization_code',
                 'code': '1234',
-            }) == request.text
```

Comment:
`text` came from `_RequestObjectProxy`, which is the actual type passed. It doesn't exist on `Request`.